docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -121,6 +121,31 @@ Connect will restore connections to all resources that were active before you cl
 When restoring terminal tabs, Teleport Connect doesn't attempt to re-execute commands that were in
 progress when the app was closed. It will only restore the working directory for those tabs.
 
+## Using tsh outside of Teleport Connect
+
+Teleport Connect ships with its own bundled version of tsh. Connect will always use this version of
+tsh for any actions performed within the app.
+
+Connect makes tsh available to use in your terminal of choice as well, with some slight differences
```

Comment:
This is the first time we've said "Connect" instead of "Teleport Connect" - was this intentional? I'm inclined to stick to the official product name.

Reply:
> This is the first time we've said "Connect" instead of "Teleport Connect" - was this intentional? I'm inclined to stick to the official product name.

It was semi-intentional as I got tired of repeating "Teleport Connect" over and over again but I agree that spelling out the full name looks better.